README.md

```diff
@@ -172,6 +172,29 @@ You can configure the mapper, by going to **Clients** > ***your-client-name*** >
 
 ![mapper](./docs/assets/mapper.png)
 
+### Events
```

Comment:
Header of this table should be "Admin Events" so we can eventually add a second table for "Login Events".

We should also put a relative link to this section in the "Contents" section above.

Reply:
Done